Source/ProofGeneration/BoogieIsaInterface/MembershipLemmaManager.cs

```diff
@@ -148,6 +149,52 @@ string theoryName
             AddDisjointnessLemmas(GlobalsMaxLocalsMin.Item1, GlobalsMaxLocalsMin.Item2);
             AddWellFormednessLemmas();
         }
+        
+        public MembershipLemmaManager(
+            IsaProgramGeneratorConfig config,
+            IsaProgramRepr isaProgramRepr,
+            IsaBigBlockInfo isaBigBlockInfo,
+            Tuple<int, int> GlobalsMaxLocalsMin,
+            IVariableTranslationFactory factory,
+            string theoryName
+        )
```

Comment:
this constructor seems to share a lot of code with the previous one. It would be good to share that code.